config/server-vars.yml

```diff
@@ -23,16 +23,70 @@ EDXAPP_PREVIEW_LMS_BASE: "{{ EDXAPP_LMS_BASE }}"
 #EDXAPP_ENABLE_THIRD_PARY_AUTH: true
 #EDXAPP_ENABLE_OAUTH2_PROVIDER: true
 
+edxapp_theme_name: 'default'

```

Comment:
See task "checkout theme" in [edxapp/tasks/deploy.yml](https://github.com/edx/configuration/blob/named-release/dogwood.rc/playbooks/roles/edxapp/tasks/deploy.yml). Open edX playbooks may already support clone/checkout of stanford theme repo using just config 'edxapp_theme_source_repo' and 'edxapp_theme_version'.